orion.py

```diff
@@ -85,6 +85,8 @@ def orion(**kwargs):
         else:
             uuids = [uuid for uuid in re.split(' |,',baseline) if uuid]
             uuids.append(uuid)
+            buildUrls = orion_funcs.get_build_urls(uuids,match)
```

Comment:
sorry. forgot noticing this earlier. In the corresponding `if` block of this `else` block we already got this logic covered. We can simply move it out of the `if-else` block's scope.